source/tests/unit/ni_fake_service_tests.cpp

```diff
@@ -117,6 +117,33 @@ TEST(NiFakeServiceTests, NiFakeService_InitWithOptionsAndResetServer_SessionIsCl
   EXPECT_EQ(0, session_repository.access_session(session.id(), ""));
 }
 
+TEST(NiFakeServiceTests, NiFakeService_InitExtCalAndResetServer_SessionIsClosed)
```

Comment:
Note: This test is similar to NiFakeService_InitWithOptionsAndResetServer_SessionIsClosed test found above